sqldelight-gradle-plugin/src/test/kotlin/com/squareup/sqldelight/GradleRunner.kt

```diff
@@ -0,0 +1,22 @@
+package com.squareup.sqldelight
+
+import org.gradle.testkit.runner.GradleRunner
+import java.io.File
+
+internal fun GradleRunner.withCommonConfiguration(projectRoot: File): GradleRunner {
```

Comment:
heres the actual interesting thing, everything else is just removing some duplicate code in favor of this